apps/dash-covid-xray/assets/style-covid.css

```diff
@@ -1,56 +1,18 @@
-/* make the graph still visible while the loading spinner is displayed */
-*[data-dash-is-loading="true"]{
-    visibility: visible;
+.card {
+    margin-top: 1em;
+    height: 50vh;
```

Comment:
```suggestion
    height: 60vh;
```

Reply:
so that figures a bit bigger (I made some quick tests on my laptop and with 60 you can see the 4 figures on one screen, not all the text but for this you can scroll a bit).